content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -2,143 +2,337 @@
 title: Easily Create and Manage AWS EKS Kubernetes Clusters
 h1: "Easily Create and Manage AWS EKS Kubernetes Clusters with Pulumi"
 date: "2019-01-24"
-meta_desc: "Use Pulumi to easily deploy new clusters, manage AWS resources, and deploy Kubernetes apps. See how to provision an EKS cluster with a CLI single gesture."
+meta_desc: "Use Pulumi to easily deploy new clusters, manage AWS resources, and deploy Kubernetes apps. See how to provision an EKS cluster with a single CLI command."
 meta_image: "easy-eks.png"
-authors: ["luke-hoban"]
-tags: ["AWS","Kubernetes"]
+authors: 
+  - luke-hoban
+  - levi-blackstone
+tags:
+  - aws
+  - kubernetes
 ---
 
-
 Provisioning, managing, and monitoring a Kubernetes cluster is
-no easy task. AWS now offers EKS to ease a lot of that burden -- but
-it's still difficult to get up and running. Pulumi's infrastructure as
-code to the rescue! We can now provision an entire EKS cluster with a
-CLI single gesture, thanks to [the `@pulumi/eks`
-package](https://github.com/pulumi/pulumi-eks). Let's see how.
+no easy task. AWS now offers EKS to ease a lot of that burden -- but
+it's still difficult to get up and running. Pulumi's infrastructure as
+code to the rescue! We can now provision an entire EKS cluster with a
+single CLI command, thanks to [the `@pulumi/eks`
+package](https://github.com/pulumi/pulumi-eks). Let's see how.
 <!--more-->
 
+TODO: update this image?
 ![Pulumi making EKS Easy](./easy-eks.png)
 
+{{% notes type="info" %}}
+*Update (December 2020)*
```

Comment:
Is there a way to get this to render with the ℹ and the "Update (December 2020)" on the same line? Would look a bit neater that way, but if not no worries.